terraform/modules/pmap-service/variables.tf

```diff
@@ -73,3 +73,17 @@ variable "gcs_events_filter" {
   type        = string
   default     = null
 }
+
+variable "pmap_resource_scope" {
+  description = <<EOT
+        "This is the scope for where the resources resides in.
+        Options can be one of the following:
+        projects/{PROJECT_ID}
+        projects/{PROJECT_NUMBER}
```

Comment:
What is the difference between project id and number, did you mean project name? Should we accept name? I remember an issue last week was related to the scope was project name.

Reply:
It's intended to be project/number and project/id. 

You can refer to this: https://cloud.google.com/asset-inventory/docs/reference/rest/v1/TopLevel/searchAllResources#path-parameters. These are the acceptable parameters allowed by the API for searching resources.

Reply:
Got it, thanks!